scala-generator/src/main/scala/models/generator/ScalaClientMethodConfig.scala

```diff
@@ -199,6 +203,15 @@ private lazy val defaultAsyncHttpClient = PooledHttp1Client()
     def generateCirceJsonOf(datatypeName: String): String = s"org.http4s.circe.jsonOf[$datatypeName]"
     def generateCirceJsonEncoderOf(datatypeName: String): String = s"org.http4s.circe.jsonEncoderOf[$datatypeName]"
     def serverImports: String = ""
+    def routeKind: String = "trait"
+    val asyncTypeImport: String = "import cats.effect._"
+    def routeExtends: Option[String] = None
+    def clientImports: String = """import org.http4s.client.blaze._
```

Comment:
Should be no need for concrete `blaze` import - everything can be done in terms of `org.http4s.client`.

Reply:
this is for the legacy http4s < 0.18, it's overridden in 0.18

Reply:
👍